integrations/jira/.port/resources/port-app-config.yaml

```diff
@@ -39,3 +37,24 @@ resources:
             project: .fields.project.key
             parentIssue: .fields.parent.key
             subtasks: .fields.subtasks | map(.key)
+            assignee: .fields.assignee.accountId
+            reporter: .fields.reporter.accountId
```

Comment:
what will be the behaviour when a ticket doesn't have assignee details. Will it stop the entity from being upserted

Reply:
No, the entity is created and the property is empty